robottelo/ui/computeresource.py

```diff
@@ -137,10 +135,107 @@ def update(self, name, newname=None, parameter_list=None,
         self.click(common_locators['submit'])
 
     def delete(self, name, really=True):
-        """Removes the compute resource entity"""
+        """Removes the compute resource entity."""
         self.delete_entity(
             name,
             really,
             locators['resource.delete'],
             drop_locator=locators['resource.dropdown'],
         )
+
+    def access_compute_resource(self, res_name):
```

Comment:
not sure why we need specific method for this operation. Doesn't `self.click(self.search(res_name))` serve the same purpose?